server/src/main/java/io/druid/server/coordinator/SegmentProcessingThrottler.java

```diff
@@ -28,107 +28,94 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 /**
- * The ReplicationThrottler is used to throttle the number of replicants that are created and destroyed.
+ * The SegmentProcessingThrottler is used to throttle the number of segments that are loaded, replicated and destroyed.
  */
-public class ReplicationThrottler
+public class SegmentProcessingThrottler
```

Comment:
the coordinator run loop is single threaded at present. 
Added a comment about thread safety on top of this class.